completion/available/system.completion.bash

```diff
@@ -15,11 +25,37 @@ elif [[ -r /etc/profile.d/bash_completion.sh ]] ; then
   source /etc/profile.d/bash_completion.sh
 
 elif [[ $OSTYPE == 'darwin'* ]] && _command_exists brew ; then
-  BREW_PREFIX=${BREW_PREFIX:-$(brew --prefix)}
+  : ${BREW_PREFIX:-$(brew --prefix)}
```

Comment:
this confilicts with #1910 isnt it?
we should decide what should we merge first (I opt for #1910)

Reply:
It's easy enough to rebase small changes like this. I'd rather get what can be merged quickly so I can just rebase one way or the other, so either way works for me!